quickannotator/api/v1/utils/shared_crud.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from sqlalchemy.orm import aliased, sessionmaker, Session, Query, DeclarativeBase
+import quickannotator.db as qadb
+from quickannotator.db import build_annotation_table_name, create_dynamic_model
+import shapely
+import json
+from sqlalchemy import func, text
+
+
+# TODO: Remove session from params
+def get_tile(session: Session, annotation_class_id: int, image_id: int, tile_id: int) -> qadb.Tile:
+    result = session.query(qadb.Tile).filter_by(
+        annotation_class_id=annotation_class_id,
+        image_id=image_id,
+        tile_id=tile_id
+    ).first()
+    return result
+
+
+def compute_custom_metrics() -> dict:
+    return {"iou": 0.5}
+
+# TODO: Remove session from params
+def insert_new_annotation(session, image_id, annotation_class_id, is_gt, tile_id, polygon: shapely.geometry.Polygon):
```

Comment:
note how here you have a different odering of parameters as compared to above, without any really strong justification for doing so. this is mainly because as humans if we don't have strict rules, we will act fairly arbitrarily. this is easy for us, but you can imaigne if you're trying to work on this project having different orderings can be confusing and lead to unintended errors - thus its better to make a rule and stick with it, so that you don't actually _have_ to think about what to do, and instead just do the right thing directly the first time. see what i mean?